.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 example
 index.html
 vite.config.mts
+package-lock.json
```

Comment:
Looks like this repo is using pnpm, 
Since we also using pnpm in AMv2 project, can we just keep using the same tool?

Reply:
Yes, we can keep using pnpm 👌 
Do you mean we should keep `package-lock.json` when using pnpm?

Reply:
if we use pnpm, we shouldn't have `package-lock.json` in this project, so we don't need to update `.gitignore` file.

Reply:
Ooohh, I get it now, thanks!
Let me remove this then, yes!

Reply:
Thanks for the catch, removed in https://github.com/smartnews/antd-img-crop/pull/1/commits/c6b639cfd70478c02d05ecf942afd46ec02808d4 🙇